packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testTruncate.st

```diff
@@ -0,0 +1,9 @@
+testing
+testTruncate
+
+	| scrollPane |
+	instance bounds: (50 @ 50 corner: 200 @ 200).
+	scrollPane := SScrollPane forMorph: instance.
+	instance truncateDiagram.
+
+	self assert: (instance boundsInWorld containsRect: scrollPane boundsInWorld).
```

Comment:
okay, then you could rename the test to `testTruncateOfEmptyDiagram`or something like this. Do you write a second test for the case with a non empty scrollpane?